test/query.test.js

```diff
@@ -19,4 +20,16 @@ describe('Testing query route', () => {
     expect(searchTerm.length).toBeGreaterThan(256);
     expect(res.status).toBe(400);
   });
+
+  test('Testing with search() returning only id', async () => {
+    await indexPost('First test data is going to be posted.', 1000);
+    await indexPost('Second real data is going to be posted.', 1001);
+    await indexPost('Third test data is going to be posted.', 1002);
+    await indexPost('Fource test data is going to be posted.', 1003);
+    const elasticPosts = await search('test');
+    expect(elasticPosts.values.length).toBe(3);
+    expect(Object.keys(elasticPosts.values[0]).length).toBe(1);
+    expect(Object.keys(elasticPosts.values[0])).toEqual(expect.arrayContaining(['id']));
+    expect(Object.keys(elasticPosts.values[1])).toEqual(expect.arrayContaining(['id']));
```

Comment:
Can you remove these tests? but move line 32 and 33 to `feed.test.js?`

Reply:
Thank you for good advice. I did it.